cache/redis/mock.go

```diff
@@ -0,0 +1,54 @@
+package redis
```

Comment:
first of all this is not a mock it is a stub.
What is the usage for this?
It is just a cache implementation of a map...

Reply:
Its use was to provide a different way to mock (yes, indeed this is a stub), Redis, other than introducing a new third-party dependency (miniredis). I'd like to hear your opinion on that, on whether it's worth it.

Reply:
Tests with miniredis tests tha our code works with miniredis... The goal is to know that it works with redis... These are integrations tests. We have to wait for the ticket about integration tests #142 in order to create integrations tests here.